docs/features/settings.md

```diff
@@ -62,7 +62,7 @@ Skip first LEDs | 0–length | 0 | Will turn off the first one or more LEDs and
 Off Refresh | on/off | off (typically) | WLED doesn't send out data if all of its outputs are off, but some pixels (notably TM1814) will go into a demo mode after a period of inactivity, and setting forces WLED to periodically send out additional "off" commands [*only appears if "Type" is set to a type that supports multiple pixels; default is "on" if "Type" is set to "TM1814"*]
 Inverted output | on/off | off | Invert the output's state (i.e. if the output is bright when it's supposed to be dark, set this to "on") [*only appears if "Type" is set to a type that supports output inversion*]
 IP address | IPv4 | (blank) | Set the IP address where the output data should be sent to [*only appears if "Type" is set to a type that supports network output*]
-Auto-calculate white channel from RGB | multiple options | "None" | Selects whether WLED should attempted to generate white-channel information for colors that are only defined as red, green, and blue values [*only appears if "Type" is set to a type that has more than three color channels*]
+Auto-calculate white channel from RGB | multiple options | "None" | Selects whether WLED should attempted to generate white-channel information for colors that are only defined as red, green, and blue values [*only appears if "Type" is set to a type that has more than three color channelsl or to "PCM White"*]
```

Comment:
Not exactly correct, unfortunately.
The correct wording should be "... type that has a white channel included, including a single PWM White."

One could also add: "NOTE: Setting this to anything but None will treat your output as a RGB output."